### PR TITLE
Bump bokeh requirements to 3.4.0rc2

### PR DESCRIPTION
**Pull request by philippjfr**
- [x] Fix JupyterLite test failures
- [ ] Fix auth test failures

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -248,7 +248,7 @@ def run(self):
     'setuptools >=42',
     'requests',
     'packaging',
-    'bokeh ==3.4.0rc1',
+    'bokeh >=3.4.0rc2,<3.5.0',
```
Comment (Hoxbro) [inline setup.py:251]: I think this will pull in Bokeh 3.4.0.dev8

Comment (Hoxbro) [reply setup.py:251]: It did 
image

Comment (philippjfr) [reply setup.py:251]: On conda?

Comment (philippjfr) [reply setup.py:251]: 😢

Comment (Hoxbro) [reply setup.py:251]: Yes, because of conda.

**Comment by philippjfr** [comment]
Not sure what's up with the auth errors yet but this is a big improvement, so merging.

**Pull request merged**